pkg/s3/s3.go

```diff
@@ -28,3 +57,185 @@ func GetConfig() *aws.Config {
 
 	return s3Config
 }
+
+// CreateS3BucketUsingAPod is helpful when trying to hit a cluster s3 service using the CLI since that could be used outside the cluster, or due to firewall restrictions
+func CreateS3BucketUsingAPod(ctx context.Context, clientset kubernetes.Interface, podOptions S3OpsPodOptions) error {
+	command := []string{"/s3-bucket-create.sh"}
+	pod, err := s3BucketPod(clientset, podOptions, command)
+	if err != nil {
+		return errors.Wrap(err, "failed to get pod resource")
+	}
+
+	createBucketPod, err := clientset.CoreV1().Pods(podOptions.Namespace).Create(ctx, pod, metav1.CreateOptions{})
+	if err != nil {
+		return errors.Wrap(err, "failed to create pod")
+	}
+
+	if err := k8sutil.WaitForPodCompleted(ctx, clientset, podOptions.Namespace, createBucketPod.Name, time.Minute*2); err != nil {
```

Comment:
WaitForPodCompleted  only checks for success, and If pod fail, this will pause for 2 minutes.  You could also use the Follow option in `GetPodLogs`, then you don't need to wait for pod to terminate.  But you do need to wait for it to be created.